dwave/system/composites/tiling.py

```diff
@@ -43,6 +45,12 @@ class TilingComposite(dimod.Composite, dimod.Structured, dimod.Sampler):
     graph of dimensions ``sub_m``, ``sub_n``, ``t``, or minor-embeddable to
     such a graph.
 
+    .. deprecated:: 1.32.0
+        Tiling is generalized in :class:`dwave.system.ParallelEmbeddingComposite`
+        to handle bigger and non-Chimera source graphs on any structured graph
+        supported by :class:`dwave.system.DWaveSampler` (including Zephyr).
+        ``TilingComposite`` will be removed in dwave-system 2.0.
```

Comment:
```suggestion
        Tiling is generalized in
        :class:`~dwave.system.composites.ParallelEmbeddingComposite`
        to handle bigger and non-Chimera source graphs on any structured graph
        supported by :class:`~dwave.system.samplers.DWaveSampler` (including
        Zephyr). ``TilingComposite`` will be removed in dwave-system 2.0.
```

This will create hyperlinks that work